src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateApplicant2NoticeOfProceedings.java

```diff
@@ -76,14 +80,43 @@ public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> cas
 
         if (isSoleApplication) {
             caseData.setCaseInvite(caseData.getCaseInvite().generateAccessCode());
-            generateSoleNoticeOfProceedings(caseData, caseId);
+            ReissueOption reissueOption = caseDetails.getData().getApplication().getReissueOption();
+            if (YES.equals(caseDetails.getData().getIsJudicialSeparation())) {
+                if (!DIGITAL_AOS.equals(reissueOption)) {
+                    generateSoleJSNoticeOfProceedings(caseData, caseId);
```

Comment:
can these two if statements be combined ?

Reply:
no - because I dont want it to go to the else part of line 89 if its a JS, but reissue = DIGITAL_AOS